lib/message.rb

```diff
@@ -0,0 +1,21 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv/load'
+class Message 
+
+  def initialize
+    @account_sid = ENV['TWILIO_ACCOUNT_SID']
```

Comment:
Good use of environment variables.
Injecting the Twilio class would have made testing easier and I would encourage you to try this approach.